src/cljs/athens/events.cljs

```diff
@@ -815,43 +815,35 @@
                                    (. -firstChild)
                                    (.getAttribute "data-uid"))
                            uid)
-        [uid embed-id]  (db/uid-and-embed-id uid)
-        block           (db/get-block [:block/uid uid])
-        {:block/keys    [children order] :or {children []}} block
-        parent          (db/get-parent [:block/uid uid])
```

Comment:
Obs: At some point we should remove the original fns in `db/`, once their references are all moved to `common-db/`. But `carve` should tell us when we can remove them.